src/Components/OATPropertyEditor/Modal.tsx

```diff
@@ -8,7 +8,7 @@ import { IOATEditorState } from '../../Pages/OATEditorPage/OATEditorPage.types';
 import FormRootModelDetails from './FormRootModelDetails';
 
 export enum FromBody {
-    property = 'Property',
+    property = 'formProperty',
```

Comment:
![image](https://user-images.githubusercontent.com/97867829/173451832-9a9e594f-bf9b-4207-8dbc-cb301a2118a2.png)
I actually made the change because I found an error the enum was waiting for a Property in the modalBody but it looks like @miltonzarmada change that in the component using the Modal a couple months ago and was generating an error loading the Modal. Its actually trying to fix this error reported in the April comments "3-The entire world blew up when I clicked the Info icon next to this" now I just fixed the Modal so it load with the content its being called now I'll Ping Milton to check the logic before that import and fix the name to FormProperty everywhere

Reply:
This might already be on staging

Reply:
Merged Staging into PR branch and addressed the issue